js/app/modules/contentGroup/mediaLightbox.js

```diff
@@ -57,6 +57,8 @@ const MediaLightbox = new Module.Class({
     },
 
     _on_close: function () {
+        this.lightbox_widget.destroy();
```

Comment:
Yea I tried that, and still it would play in the background after closing lightbox